src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  let total;
+
+  if (days < 3) {
+    total = dailyRent * days;
```

Comment:
These can be return statements for slightly faster code execution

Reply:
Done!